.github/workflows/python-deps.yml

```diff
@@ -27,11 +27,22 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      - name: Install prerequisites
+        # brew can be finicky but it does not always means that the rest of the job will fail
+        run: |
+          # Needed for xmlsec
+          brew update -q && brew install libxml2 libxmlsec1 pkg-config || exit 0
+
       - name: Setup Python
         uses: ./.github/actions/setup-python
+        with:
+          # Non-default value since install-py-dev installs all requirements files
+          # We also want to bust the cache if the action and workflow change
+          cache-files-hash: ${{ hashFiles('requirements-*.txt', '.github/actions/setup-python/action.yml', '.github/workflows/python-deps.yml') }}
 
       - name: Install dependencies
         run: |
           python -m venv .venv
           source .venv/bin/activate
           make install-py-dev
+          pip install -r requirements-pre-commit.txt
```

Comment:
I also want those requirements to be tested.

Reply:
Should we go one step further and test `make setup-git`?

Reply:
Sure.